app/forms/field_handlers/field_handler.py

```diff
@@ -66,20 +84,14 @@ def get_mandatory_validator(self) -> Union[ResponseRequired, Optional[Any]]:
             )
         return validators.Optional()
 
-    def get_schema_value(self, schema_element: dict) -> Optional[AnswerValueTypes]:
+    def get_schema_value(
+        self, schema_element: dict
+    ) -> Union[ValueSourceEscapedTypes, ValueSourceTypes]:
```

Comment:
Not a blocker, but we could make an alias for this called something like `ResolvedValueSourceTypes`. This is used in multiple places and in future we will use it in other places and since we are using a custom alias already, using one would make it clearer in my opinion.

Reply:
:laughing: You introduced it